getsolus/packages
# [RFC]zeek: initial inclusion at 6.0.2 (replacing bro)

**Summary**

Initial inclusion of zeek

**Test Plan**

- Run `zeek` file.zeek and observe similiar result with `bro`
- Run a few example provided on [here](https://try.zeek.org)

**Checklist**


**Packager's Comment**

- Build with `--disable-abi-report`, because it goes to loop and takes up to 3GB of RAM.
- `zkg` is not built  because missing dependency (GitPython).
- `trace-summary` is non-functional because missing dependency (ipsumdump) and no build flag to disable it.
- Consider putting `needs mantainer` label or depecrate this package as it is pretty complex to maintain and not exactly for average desktop user.
- I don't feel like putting anymore effort into this package, someone can just merge it, close it or commandeer it from now on.
